index.html

```diff
@@ -200,6 +200,48 @@ <h2>About Myself</h2>
           />
         </ul>
       </section>
+      <section class="contactme">
+        <h1 class="contacttxt font">Contact me</h1>
+        <p class="paragraph font">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form
+          id="form-id"
+          action="https://formspree.io/f/meqwgnyz"
+          method="post"
+        >
+          <input
+            type="text"
+            name="name"
+            id="form-name"
+            maxlength="30"
+            placeholder="Enter your name"
+            class="inputtxt1 radius"
+          />
+          <input
+            name="Email"
+            id="email-id"
+            type="email"
+            placeholder="Enter your email address"
+            class="inputtxt2 radius"
+          />
+          <div id="errormessage"></div>
+          <textarea
+            name="textarea"
+            id="form-message"
+            placeholder="Write your message here"
+            cols="30"
+            rows="10"
+            class="inputtxt3 radius"
+            maxlength="500"
+          ></textarea>
```

Comment:
- [ ] Please make sure that the user can not submit the form if any of the inputs is empty, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=A%20user%20cannot%20submit%20the%20form%20if%20one%20or%20more%20of%20the%20fields%20is%20empty.) requirement. You can use `required` to achieve this.

Reply:
Done